rules/policy.go

```diff
@@ -535,6 +550,8 @@ func appendProtocolMatch(match iptables.MatchCriteria, protocol *proto.Protocol,
 	return match
 }
 
+var SkipRule = errors.New("Rule skipped")
```

Comment:
Any reason to move this?   If not, please put it back to reduce the probability of conflicts with later merges/cherry-picks.